src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/ApplicationService.kt

```diff
@@ -117,6 +117,7 @@ class ApplicationService(
         reservedSlot = isReservedSlot,
         visitType = sessionTemplate.visitType,
         restriction = createApplicationDto.applicationRestriction.getVisitRestriction(),
+        userType = createApplicationDto.userType,
```

Comment:
also does eventAudit need to change as a result. Can be a a part of a different story.

Reply:
This needs to be done,.. but not for now